rwengine/include/data/VehicleGenerator.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef RWENGINE_VEHICLEGENERATOR_HPP
+#define RWENGINE_VEHICLEGENERATOR_HPP
+
+/**
+ * Stores information about where the game can generate vehicles.
+ */
+struct VehicleGenerator
+{
+	glm::vec3 position;
+	float heading;
+	/** ID of the vehicle to spawn, or -1 for random. */
+	int vehicleID;
+	int colourFG;
+	int colourBG;
+	bool alwaysSpawn;
+	short alarmThreshold;
+	short lockedThreshold;
+
+	int minDelay;
+	int maxDelay;
+	/** Incrementing timer (in ms), will only spawn a vehicle when minDelay < spawnTimer and will always spawn if maxDelay < spawnTimer */
+	int lastSpawnTime;
```

Comment:
`maxDelay` is not checked yet! This makes the comment wrong.
We need a `/// @todo` here.